examples/audio/spatial_audio_2d.rs

```diff
@@ -0,0 +1,94 @@
+//! This example illustrates how to load and play an audio file, and control where the sounds seems to come from.
+use bevy::prelude::*;
+
+fn main() {
+    App::new()
+        .add_plugins(DefaultPlugins)
+        .add_startup_system(setup)
+        .add_system(update_positions)
+        .run();
+}
+
+fn setup(
+    mut commands: Commands,
+    asset_server: Res<AssetServer>,
+    audio: Res<Audio>,
+    audio_sinks: Res<Assets<SpatialAudioSink>>,
+) {
+    let gap = 4.0;
+
+    let music = asset_server.load("sounds/Windless Slopes.ogg");
+    let handle = audio_sinks.get_handle(audio.play_spatial_with_settings(
+        music,
+        PlaybackSettings::LOOP,
+        Transform::IDENTITY,
+        gap,
+        Vec3::ZERO,
+    ));
+    commands.insert_resource(AudioController(handle));
+
+    // Putting the visual presentation in a parent with a big scale as sound attenuation happens quite fast so distance are kept small
+    commands
+        .spawn_bundle(SpatialBundle {
+            transform: Transform::from_scale(Vec3::splat(100.0)),
+            ..default()
+        })
+        .with_children(|parent| {
+            // left ear
+            parent.spawn_bundle(SpriteBundle {
+                sprite: Sprite {
+                    color: Color::RED,
+                    custom_size: Some(Vec2::splat(0.2)),
+                    ..default()
+                },
+                transform: Transform::from_xyz(-gap / 2.0, 0.0, 0.0),
+                ..default()
+            });
+
+            // right ear
+            parent.spawn_bundle(SpriteBundle {
```

Comment:
Haha for me I'm getting the opposite of others: this is flipped / incorrect for me in release (green is left) and works as expected in debug (green is right).